cmake/Utils.cmake

```diff
@@ -244,7 +244,11 @@ function(caffe2_binary_target target_name_or_src)
   endif()
   add_executable(${__target} ${__srcs})
   add_dependencies(${__target} ${Caffe2_MAIN_LIBS_ORDER})
-  target_link_libraries(${__target} ${Caffe2_MAIN_LIBS} ${Caffe2_DEPENDENCY_LIBS})
+  if (USE_CUDA)
```

Comment:
I assume this is no longer needed with the interface library we have?

Reply:
It actually was on Windows

Reply:
nvm, saw your note above.